configure.ac

```diff
@@ -220,12 +220,15 @@ Makefile
 gstd/Makefile
 gst_client/Makefile
 libgstc/Makefile
+libgstc/c/Makefile
+libgstc/python/Makefile
```

Comment:
we need a Makefile for JS to tell autotools that the js file is part of the project. Otherwise it doesn’t get include in the dist tarballs.